### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_ActivityBar_Keyboard: Focus is not present on expanded "setting button" items list.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52385]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Keyboard,Usability

**Repro Steps:**
1.Navigate to "Activity Bar" at left side of the screen after login.
2.Navigate to "SettingsButton" and click on it.
3.Observed that after expanding the "SettingsButton" the keyboard focus is not present on any item list of expanded "Settings".

**Actual:**
After expanding the settings button user have to click on arrow keys then only keyboard focus is visible on the opened expanded item list.

**Expected:**
When setting button get clicked then, keyboard focus should be visible on item list.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets.
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
If keyboard focus is not visible then user will not know where the current keyboard focus is.

**Attachment For Refernce:**
[[Accessibility][Usability]A11y_VSCode_ActivityBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2117987/Accessibility.Usability.A11y_VSCode_ActivityBar_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
This will change with @sbatten work

**Comment by sbatten** [comment]
This is fixed with the new menu widgets
